src/nanoarrow/common/array_test.cc

```diff
@@ -1555,6 +1555,149 @@ TEST(ArrayTest, ArrayTestAppendToLargeListArray) {
 #endif
 }
 
+TEST(ArrayTest, ArrayTestAppendToListViewArray) {
```

Comment:
These tests were copy/paste from their list counterparts; in a follow up commit or PR I'd like to try and parametrize these